### ISSUE TITLE
RSPEC-5730 Allowing mixed-content is security-sensitive

### ISSUE THREAD

**Issue by eric-therond-sonarsource** [SonarSource/SonarJS#1950]
Implements [RSPEC-5732](https://jira.sonarsource.com/browse/RSPEC-5730) MMF-1898

For both JS/TS

Should raise on the csp()/helmet() call (depending on the `helmet-csp`/`helmet` module used) if the `blockAllMixedContent` ("block-all-mixed-content" is an alias) directive is missing:

```
var app = express()

  app.use(csp({
    directives: {
      defaultSrc: ["'self'", 'example.com', 'code.jquery.com'],
    } // Sensitive: blockAllMixedContent is not set
  }));
```
Code examples: https://github.com/SonarSource/security-expected-issues/tree/master/javascript/express.js/RSPEC-5730

when `contentSecurityPolicy` property when set to `false`  NO NEED TO RAISE, because [rule RSPEC-5728](https://github.com/SonarSource/SonarJS/issues/1949) will be already triggered::

```
app.use(
    helmet({
      contentSecurityPolicy: false, // sensitive for rule S5728 only
    })
  );
```

**Comment by eric-therond-sonarsource** [comment]
Reopened because:
* the message is incorrect/not formatted ([see for instance this issue](https://peach.sonarsource.com/security_hotspots?id=security-expected-issues-express-js_nodev8&hotspots=AXWxbigqMh1ExG5XrVRH))
* we have 1 TP/1 FN out of [two test cases on security-expected-issues](https://github.com/SonarSource/security-expected-issues/blob/master/javascript/express.js/RSPEC-5730/sensitive.js):
```
  app.use(
    csp({
      directives: {
        defaultSrc: ["'self'", 'example.com', 'code.jquery.com'],
      },
    }),
  ); // TP: helmet-csp is used but the blockAllMixedContent directive is missing

  // other sensitive case

  app.use(
    helmet.contentSecurityPolicy({
      directives: {
        'default-src': ["'self'", 'example.com', 'code.jquery.com'], // FN: blockAllMixedContent directive is missing
      },
    }),
  );
```
